portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -35,10 +41,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     if (userService.isUserLoggedIn()) {
       final String userEmail = userService.getCurrentUser().getEmail();
       final String urlToRedirectToAfterUserLogsOut = "/";
-      final String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      final String logoutUrl = 
+        userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      final String id = userService.getCurrentUser().getUserId();
+      final String username = getUsername(userEmail, id);
```

Comment:
So it looks like you added all these finals in a previous CL in response to my comment (https://github.com/zpChris/step/pull/18/commits/a33b6024c1ea67d20fcfbf10caf173a5a14edd4b). What i'd meant was that the urls should be class constants, and that's my bad for just giving approval and not requesting changes. Final variables are a great tool, but they should really only be used when necessary. Could you remove the 'final' keyword from all these variables?

Reply:
Done!